pkg/controller/storagecluster/cephcluster.go

```diff
@@ -422,7 +444,7 @@ func newStorageClassDeviceSets(sc *ocsv1.StorageCluster, serverVersion *version.
 			// enable the new behavior while the console is waiting
 			// to be updated.
 			// TODO: Remove this behavior when OCP console is updated
-			count = count / 3
+			count = count / replica
```

Comment:
Just highlighting this hunk to @jarrpa 's attention - this was discussed very controversally before, e.g. #835  ... :smile:

Reply:
Yes, don't do this. We want to freeze this behavior as hard as possible.

Reply:
Done